priv/static/phoenix_live_view.js

```diff
@@ -326,6 +326,10 @@ var LiveView = (() => {
       return wantsNewTab || e.target.getAttribute("target") === "_blank";
     },
     isUnloadableFormSubmit(e) {
+      var _a, _b;
+      if (((_a = e.target) == null ? void 0 : _a.getAttribute("method")) === "dialog" || ((_b = e == null ? void 0 : e.submitter) == null ? void 0 : _b.getAttribute("formmethod")) === "dialog") {
+        return false;
+      }
```

Comment:
I don't know anything about the problem this PR is concerned with. But I noticed this conditional. It seems like it can be simplified quite a bit. I don't think the ternaries or `void 0` are necessary.

Unless I'm mistaken, it could be replaced with:

```js
if (e.getAttribute("formmethod") === "dialog" || (e.target && e.target.getAttribute("method") === "dialog"))
```

Which would be less cryptic.

Just a suggestion. 🙂